server/lib/data_import/my_anime_list.rb

```diff
@@ -31,15 +32,28 @@ def get_media(external_id) # anime/1234 or manga/1234
       end
     end
 
+    def get_character(external_id)
+      media = Mapping.lookup('myanimelist', external_id)
```

Comment:
What is being passed in already is `character/422`. That would make it `character/character/422`.